src/components/ImagerySearch.tsx

```diff
@@ -133,13 +156,14 @@ export class ImagerySearch extends React.Component<Props, {}> {
   //
 
   private dateValidation() {
-      return moment(this.props.dateFrom).isSameOrBefore(this.props.dateTo)
+    const from = moment.utc(this.props.dateFrom, DATE_FORMAT, true)
```

Comment:
I assume the user enters dates as UTC and they just know to use UTC?

Reply:
But this should be UTC.